internal/mobcommands/pathto.go

```diff
@@ -0,0 +1,48 @@
+package mobcommands
+
+import (
+	"strconv"
+	"strings"
+
+	"github.com/GoMudEngine/GoMud/internal/mapper"
+	"github.com/GoMudEngine/GoMud/internal/mobs"
+	"github.com/GoMudEngine/GoMud/internal/rooms"
+	"github.com/GoMudEngine/GoMud/internal/util"
+)
+
+func Pathto(rest string, mob *mobs.Mob, room *rooms.Room) (bool, error) {
```

Comment:
Thoughts on a issue to add docblocks on our command functions? Could be nice if we start autogenerating code docs or for others coming into the code.

Reply:
We can come up with a standard practice, but I think we should be idiomatic about it: https://tip.golang.org/doc/comment

docblocks/javadoc is nice, but not idiomatic, and we can leverage `pkt.go.dev` if we follow their idioms (this link obviously isn't using them, but you get the idea):

https://pkg.go.dev/github.com/GoMudEngine/GoMud@v0.0.0-20250425175142-d2441433676d/modules/auctions